tick/robust/tests/serializing_test.py

```diff
@@ -0,0 +1,50 @@
+
+# License: BSD 3 clause
+
+
+import io, unittest
+import numpy as np
+
+import dill as pickle
+
+from tick.base_model.tests.generalized_linear_model import TestGLM
+
+from tick.prox import ProxL1
+from tick.linear_model import ModelLinReg, SimuLinReg
+from tick.linear_model import ModelLogReg, SimuLogReg
+from tick.linear_model import ModelPoisReg, SimuPoisReg
+from tick.linear_model import ModelHinge, ModelQuadraticHinge, ModelSmoothedHinge
+
+from tick.robust import ModelAbsoluteRegression, ModelEpsilonInsensitive, ModelHuber, \
+                        ModelLinRegWithIntercepts, ModelModifiedHuber
+
+from tick.simulation import weights_sparse_gauss
+
+class Test(TestGLM):
+    def test_solver_saga(self):
+
+        model_map = {
+            ModelAbsoluteRegression : SimuLinReg,
+            ModelEpsilonInsensitive : SimuLinReg,
+            ModelHuber              : SimuLinReg,
+            # ModelLinRegWithIntercepts : SimuLinReg, # has coeffe issue?
+            ModelModifiedHuber      : SimuLogReg
+        }
+
+        for mod in model_map:
+            model = mod(fit_intercept=False)
+
+            coeffs0 = weights_sparse_gauss(20, nnz=5)
+            interc0 = None
+
+            features, labels = model_map[mod](
+                coeffs0, interc0, n_samples=3000, verbose=False,
+                seed=123).simulate()
+            model.fit(features, labels)
+
+            pickled = pickle.loads(pickle.dumps(model))
+
+            self.assertTrue(model._model.compare(pickled._model))
```

Comment:
Maybe can we also check that `model.loss(features[0]) == pickled.loss(features[0])`

Reply:
k